src/utils/string.formatter.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * String formatting functions
+ */
+const REGEX_NON_PRINTABLE_CHARS: RegExp =
```

Comment:
could this be renamed to REGEX_MATCH_NON_PRINTABLE_CHARS ? to make it clearer what the regex does

Reply:
I'd rather keep it short

Reply:
the name leads me to believe the regex is a list of non printable characters, which is the opposite of what it is. A name that makes it clear it is matching non printable characters would be ideal

Reply:
Ok renamed.